src/test/java/hudson/plugins/warnings/parser/IarParserTest.java

```diff
@@ -49,7 +49,7 @@ public void testWarningsParser() throws IOException {
 
         Iterator<FileAnnotation> iterator = warnings.iterator();
 
-        assertEquals("Wrong number of warnings detected.", 9, warnings.size());
+        assertEquals("Wrong number of warnings detected.", 8, warnings.size());
```

Comment:
This makes no sense for me. If the IAR compiler changed the format now we should support both versions. So please add a new test case (or example message in the file) that matches the capitalized version.